logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -144,7 +147,7 @@ internal class MessageSenderImpl internal constructor(
                     else Either.Left(StorageFailure.Generic(IllegalArgumentException("Client cannot send server messages")))
                 result
                     .onFailure {
-                        val type = message.content.getType()
+                        val type = getType(message.content) ?: "Unknown"
```

Comment:
Oh, I forgot about that, could you please remove this method getType (line 164) from this class as it may be affected by code minification, and use MessageContent?.getType() like it was previously?